### ISSUE TITLE
TypeError: Cannot destructure property 'Sf' of 'WebpackModules.getModule(...)' as it is undefined.

### ISSUE THREAD

**Issue by jack-glenn** [JustOptimize/return-ShowHiddenChannels#146]
I get the following error when opening Discord from your plugin.
```
TypeError: Cannot destructure property 'Sf' of 'WebpackModules.getModule(...)' as it is undefined.
    at plugin (betterdiscord://plugins/ShowHiddenChannels.plugin.js:180:17)
    at eval (betterdiscord://plugins/ShowHiddenChannels.plugin.js:1576:10)
    at eval (betterdiscord://plugins/ShowHiddenChannels.plugin.js:1577:3)
    at D.requireAddon (betterdiscord/renderer.js:5:33852)
    at D.loadAddon (betterdiscord/renderer.js:5:6322)
    at D.loadAddon (betterdiscord/renderer.js:5:31520)
    at D.loadAllAddons (betterdiscord/renderer.js:5:8814)
    at D.initialize (betterdiscord/renderer.js:5:3194)
    at D.initialize (betterdiscord/renderer.js:5:30719)
    at Object.startup (betterdiscord/renderer.js:5:385713)
```

**Comment by JustOptimize** [comment]
This has been fixed, check #140.

TLDR: Download the new version
